tap_mixpanel/streams.py

```diff
@@ -449,8 +476,10 @@ def sync(self, state, catalog, config, start_date):
                 params[self.bookmark_query_field_from] = from_date
                 params[self.bookmark_query_field_to] = to_date
 
+            if parent_data:
+                pass
             # Funnels and cohorts have a parent endpoint with parent_data and parent_id_field
-            if self.parent_path and self.parent_id_field:
+            elif self.parent_path and self.parent_id_field:
```

Comment:
I think we can remove the pass statement here and move below the if-else condition inside the first if condition.

Reply:
Removed pass statement and instead moved bellow code inside `if` condition